tensorrt_llm/_torch/model_config.py

```
@@ -295,60 +298,64 @@ def resolve_moe_backend(moe_backend: str, architecture: str) -> str:
+            # and/or a kv_cache_quant_algo that must agree with the file.
             # See tests/unittest/llmapi/test_llm_quant.py
+            extended: dict = {}
             try:
-                mixed_quant_config_file = transformers.utils.hub.cached_file(
+                ext_file = transformers.utils.hub.cached_file(
                     checkpoint_dir, 'quant_cfg.json')
-                with open(mixed_quant_config_file) as fm:
-                    json_extended_quant_configs = json.load(fm)
+                with open(ext_file) as fm:
+                    extended = json.load(fm)
             except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)